index.html

```diff
@@ -2120,588 +2138,378 @@ <h3>Thing Root Object</h3>
+  in an instance of the <a>Class</a> <code>Thing</code>
   MUST be serialized as JSON objects.
   </span></p>
 
   <p><span class="rfc2119-assertion" id="td-arrays">
   All values assigned to
   <code>links</code>, and
   <code>forms</code>
-  in an instance of <a>Class</a> <code>Thing</code>
+  in an instance of the <a>Class</a> <code>Thing</code>
   MUST be serialized as JSON arrays containing JSON objects
-  as defined in <a href="#form-serialization-json"></a>
-  and <a href="#links-serialization-json"></a>, respectively.
+  as defined in <a href="#link-serialization-json"></a>
+  and <a href="#form-serialization-json"></a>, respectively.
   </span></p>
 
   <p>
-      <span class="rfc2119-assertion" id="td-security-activation">
-          The value assigned to <code>security</code> in an instance
-          of <a>Class</a> <code>Thing</code> MUST be serialized as
-          a JSON string or as a JSON array whose elements are JSON strings.
-      </span>
+  <span class="rfc2119-assertion" id="td-security-activation">
+  The value assigned to
+  <code>security</code>
+  in an instance of <a>Class</a> <code>Thing</code>
+  MUST be serialized as JSON string or as JSON array whose elements are JSON strings.
+  </span>
   </p>
 
 </section> <!-- end of id="sec-thing-as-a-whole-json"-->
 
-<section id="property-serialization-json">
-<h3><code>properties</code></h3>
-  
-  <p>
-  The value assigned to <code>properties</code> in a <code>Thing</code> instance
-  is a <a>Map</a> of instances of <code>PropertyAffordance</code>.
-  <span class="rfc2119-assertion" id="td-properties">
-    All name-value pairs of a <a>Map</a> of <code>PropertyAffordance</code> instances
-    MUST be serialized as members of the JSON object that results from serializing the <a>Map</a>;
-    the name of a pair MUST be serialized as a JSON string and the value of the pair, an
-    instance of <code>PropertyAffordance</code>, MUST be serialized as a JSON object.
-  </span></p>
+<section id="titles-descriptions-serialization-json">
+<h3>Human-Readable Metadata</h3>
 
   <p>
-  <span class="rfc2119-assertion" id="td-property-names">
-    All name-value pairs of an instance of <code>PropertyAffordance</code>,
-    where the name is a <a>Vocabulary Term</a> included in (one of) the <a>Signatures</a> of
-    <code>PropertyAffordance</code>, <code>InteractionAffordance</code>, or <code>DataSchema</code>,
-    MUST be serialized as members of the JSON object that results from
-    serializing the <code>PropertyAffordance</code> instance, with the
-    <a>Vocabulary Term</a> as name.
-  </span>
-  See <a href="#data-schema-serialization-json" class="sec-ref"></a> for
-  details on serializing <a href="#dataschema" class="sec-ref"><code>DataSchema</code></a> instances.
+    JSON members named <code>title</code> and <code>description</code> are 
+    used within a TD document to provide human-readable metadata.
+    It can be used as comments for developers inspecting a TD document
```

Comment:
"It" -> "They" since there are two.   Also "or for display texts for a user interface"

Reply:
Fixed